pwnlib/rop/rop.py

```diff
@@ -356,6 +356,32 @@
     >>> p.sendline(b'echo hello; exit')
     >>> p.recvline()
     b'hello\n'
+
+ROP + StackRelative
+-----------------------
+
+In certain situations, you might prefer to use a stack-relative address. 
+For this purpose, you can use :class:`StackRelative` as a placeholder for a stack-relative address.
+
+    >>> context.clear(arch='amd64')
+    >>> assembly = 'pop rdi; ret; pop rsi; ret; pop rdx; ret; pop rbp; ret'
+    >>> binary = ELF.from_assembly(assembly)
+    >>> binary.symbols['funcname'] = binary.entry + 0x1000
+    >>> rop = ROP(binary)
+    >>> rop.base = 0xdead0000
+    >>> from pwnlib.rop.call import StackRelative
```

Comment:
Is this necessary in normal exploit scripts? We can add imports to the docs/ .rst for doctest code if this isn't handled by star includes.